### ISSUE TITLE
Compatibility issue with Polymerfire v0.10.3+

### ISSUE THREAD

**Issue by masonlouchart** [convoo/login-fire#43]
After an update of our Bower components, the app is not working anymore. Here is the encountered error:

`firebase-auth.html:184 Uncaught TypeError: Cannot read property 'signInWithPopup' of undefined`

We have tried with `login-fire#0.1.4` and `polymerfire#0.10.0` and it works fine.

Did you try the `login-fire` element after the [dependencies upgrade](https://github.com/convoo/login-fire/commit/2daba27e1ed04e4ed360c847684963d8b28044fa) ?

**Comment by sbeleidy** [comment]
Hi, yes we have tried it. The issue may be with data binding though.

I will test it again. Thanks for reporting the bug!

**Comment by sbeleidy** [comment]
You can see it working on [beta.webcomponents.org](https://beta.webcomponents.org/element/convoo/login-fire/demo/demo/index.html) and I just tested it locally and it seems to be working fine.

@LM450N Could you share some code and maybe I can help out?

**Comment by masonlouchart** [comment]
I created a new app for testing our sign-in flow separately and I figured out what happened. We are importing the `firebase-app` asynchronously in another element of the one that loads `login-fire`. So it's simply an import issue due to the `polymerfire` imports restructuration. `polymer-auth` has to be manually imported.

The most important point is I present you my apologies to not have do this test before open this issue. And thank you for this element.
